tfp-automation/Jenkinsfile

```diff
@@ -1,26 +1,29 @@
-pipeline {
-  agent any
-
-  stages {
+node {
```

Comment:
I don't think things break if you omit this, but all our other jenkinsfiles have the shebang `#!groovy` as the first line of each file.

Reply:
not hitting the error today, so likely encountered that error yesterday due to some syntax issue, but not reproducing it 
today!  I'll update PR to include `#!groovy` 👍